README.md

```diff
@@ -204,6 +209,19 @@ The tool also allows for selecting a folder where the files will be downloaded,
 ```
 **Note**: If needed, the user can download a selection of files from an available dataset by providing a customized `urls_list.txt` file.
 
+### Download using the download API
+
+The download API allows for downloading files from the archive and it requires the user to have access to the dataset, therefore a configuration file needs to be downloaded before starting the downloading of the files.
```

Comment:
Will be better to add a cross link to the section `### Download the configuration file`

Reply:
done